public/index.html

```diff
@@ -19,7 +19,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>PoA governance dApp</title>
```

Comment:
@vbaranov 
- we use DApp
- we use POA as a network ticker/short name
POA Network as full name

it would be good to name is `POA Network Governance DApp`
or `POA Governance DAppp`

Reply:
Ok, will do https://github.com/poanetwork/poa-dapps-voting/issues/36